src/post/sponsorship.rs

```diff
@@ -13,6 +13,7 @@ pub enum SponsorshipToken {
     Near,
     NEP141 { address: AccountId },
     USD,
+    USDT
```

Comment:
USDT is a kind of NEP141 token, so we can reuse the NEP141 (pass `NEP141 { "address": "usdt.tether-token.near" }}` instead of adding a new variant. And in this sponsorship.rs, make sure `"usdt.tether-token.near"` is allowed but right now any other address is not allowed (You can check with `if` in creation).

In frontend, map `"usdt.tether-token.near"` to `"USDT"`